components/availability/src/Availability.svelte

```diff
@@ -563,6 +570,11 @@
             <div
               class="epoch {epoch.status}"
               style="height: {epoch.height}%; top: {epoch.offset}%;"
+              aria-label="{new Date(
+                epoch.start_time,
+              ).toLocaleString()} to {new Date(
+                epoch.end_time,
+              ).toLocaleString()}; Free calendars: {epoch.available_calendars.toString()}"
```

Comment:
This information still isn't accessible. I recommend adding the list of calendars to the time slot `<button>`. If that's not easy because it's outside of the `epoch` block, then maybe a story can be created to address it later?

Reply:
It's already on the button, just as `data-available-calendars`. Will add it to the `aria-label` instead